### PR TITLE
chore: Add localization testing tool to tools directory

### PR DESCRIPTION
**Pull request by sfoslund**
#### Details

Adds a test app that can be used to test axe-windows localization to the tools directory. See the README for details on usage.

##### Motivation

Keep all tools related to axe-windows in one place

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000

### CODE REVIEW

File: tools/LocTestingSampleApp/LocTestingSampleApp.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>axe_win_testing</RootNamespace>
```
Comment (DaveTryon) [inline tools/LocTestingSampleApp/LocTestingSampleApp.csproj:6]: This doesn't match the code. Maybe
```suggestion
    <RootNamespace>AxeWinLocTesting</RootNamespace>
```

File: tools/LocTestingSampleApp/Program.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            finally
+            {
+                process.Kill();
+            }
+        }
+
+        private static Process StartTestExe()
+        {
+            var process = Process.Start(exePath);
+            Thread.Sleep(processStartupWaitTime);
```
Comment (DaveTryon) [inline tools/LocTestingSampleApp/Program.cs:50]: Timers aren't generally reliable. There's a API designed to exactly what we want:
```suggestion
            process.WaitForInputIdle();
```

File: tools/LocTestingSampleApp/Program.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            return process;
+        }
+
+        private static IScanner CreateScanner(int processId)
+        {
+            Console.WriteLine("Creating scanner...");
+            var configBuilder = Config.Builder.ForProcessId(processId);
+            configBuilder.WithOutputFileFormat(OutputFileFormat.None);
```
Comment (DaveTryon) [inline tools/LocTestingSampleApp/Program.cs:58]: These are chainable if you want:
```suggestion
            var configBuilder = Config.Builder.ForProcessId(processId)
                .WithOutputFileFormat(OutputFileFormat.None);
```

File: tools/LocTestingSampleApp/Program.cs [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        // Change this path to experiment with different test apps.
+        private static string exePath = "../WildlifeManager/WildlifeManager.exe";
+
+        // True will show full JSON scan results, false will just show a short example. The long output can
+        // be very long and confusing, since it serializes both content that should be localized (ex: rule
+        // descriptions/ text) and content that should not be localized (ex: property/field names).
+        private static bool showFullResults = false;
+
+        // Wait time to allow process to start up before beginning scan.
+        private static int processStartupWaitTime = 1000;
```
Comment (DaveTryon) [inline tools/LocTestingSampleApp/Program.cs:26]: This can go away if you use `process.WaitForIdleTime` as described elsewhere

**Review by DaveTryon — APPROVED** [review]
Very nice! I made some suggestion to consider

**Comment by sfoslund** [comment]
Sorry for the noise here, the license check kept getting stuck. Should be good now, will merge once CI goes green :)

**Pull request merged**